lib/fortnox/api/repositories/base.rb

```diff
@@ -9,9 +9,6 @@
 require_relative '../request_handling'
 
 # TODO: Temporarily disables metrics since this will be rewritten soon...
```

Comment:
Well, now that you don't disable the metrics anymore ... the comment could also be removed?